### DISCUSSION TITLE
UnityEditor.AssetDatabase 임포트

### DISCUSSION THREAD

**Discussion by Asipe42** [naverz/zepeto-studio-kor#1008] [5. Q&A]
UnityEditor.AssetDatabase 클래스를 사용하고 싶은데

아직 지원되지 않고 있나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, UnityEditor.AssetDatabase는 C#으로 구현해서 사용하셔도 됩니다.

**Reply by Asipe42** [reply]
오... 그렇군요! 답변 감사합니다~
